web/impact/impact/v1/views/organization_users_view.py

```diff
@@ -9,12 +8,14 @@
     PartnerTeamMember,
     StartupTeamMember,
 )
+from impact.v1.metadata import OrganizationUsersMetadata
 
 
 class OrganizationUsersView(APIView):
     permission_classes = (
         V1APIPermissions,
     )
+    metadata_class = OrganizationUsersMetadata
```

Comment:
Another alternative would be for the ACTIONS variable from metadata.py to be declared in this file and then have a utility function that creates the appropriate metadata class.